slices/cracklib-runtime.yaml

```diff
@@ -0,0 +1,46 @@
+package: cracklib-runtime
+
+essential:
+  - cracklib-runtime_copyright
+
+# Cracklib runs /usr/sbin/update-cracklib on install, and this
+# in turns generate the cracklib caches. This is mostly possible to do
+# as a mutation script, and is provided here:
+# https://paste.ubuntu.com/p/Jc5vvgk3tN/
```

Comment:
Nice work there! It's a shame we cannot raw bytes yet.
On another note, could this link expire some day? If so, I would want there to be something that does not expire, Github Gist maybe?